frontend/src/scenes/data-warehouse/external/DataWarehouseExternalScene.tsx

```diff
@@ -61,21 +174,73 @@ export function DataWarehouseExternalScene(): JSX.Element {
-                        'Bring your production database, revenue data, CRM contacts or any other data into PostHog.'
-                    }
-                    action={toggleSourceModal}
-                    isEmpty={shouldShowEmptyState}
-                    docsURL="https://posthog.com/docs/data/data-warehouse"
-                    productKey={ProductKey.DATA_WAREHOUSE}
-                />
-            )}
-            {!shouldShowEmptyState && <DataWarehouseTablesContainer />}
+            <div className="grid md:grid-cols-3">
+                <div className="sm:col-span-3 md:col-span-1">
+                    <DatabaseTableTree onSelectRow={selectRow} items={treeItems()} selectedRow={selectedRow} />
+                </div>
+                {selectedRow ? (
+                    <div className="px-4 py-3 col-span-2">
+                        <div className="flex flex-row justify-between items-center">
+                            <h3>{selectedRow.name}</h3>
+                            <div className="flex flex-row gap-2 justify-between">
+                                {deleteButton(selectedRow)}
+                                <Link
+                                    to={urls.insightNew(
+                                        undefined,
+                                        undefined,
+                                        JSON.stringify({
+                                            kind: NodeKind.DataTableNode,
+                                            full: true,
+                                            source: {
+                                                kind: NodeKind.HogQLQuery,
+                                                // TODO: Use `hogql` tag?
+                                                query: `SELECT ${selectedRow.columns
+                                                    .filter(({ table, fields, chain }) => !table && !fields && !chain)
+                                                    .map(({ key }) => key)} FROM ${selectedRow.name} LIMIT 100`,
+                                            },
+                                        })
+                                    )}
+                                >
+                                    <LemonButton type="primary">Query</LemonButton>
+                                </Link>
+                            </div>
+                        </div>
+                        <div className="flex flex-col">
+                            {selectedRow.external_data_source ? (
+                                <></>
+                            ) : (
+                                selectedRow.url_pattern && (
+                                    <>
+                                        <span className="card-secondary mt-2">Files URL pattern</span>
+                                        <span>{selectedRow.url_pattern}</span>
+                                    </>
+                                )
+                            )}
```

Comment:
We can probably remove the ternary and do `{!selectedRow.external_data_source && selectedRow.url_pattern && <>...</>}`